03-pipefilter.md

```diff
@@ -154,6 +154,17 @@ $ cat lengths.txt
  107  total
 ~~~
 
+> ## Output page by page {.callout}
+>
+> We'll continue to use `cat` in this lesson, for convenience and consistency,
+> but it has the disadvantage that,
+> if a file is long, you get *all* of it dumped onto your screen.
+> Probably more useful in practice is the command `less`,
+> as in `$ less lengths.txt`,
+> which displays a screenful of the file, and then stops.
```

Comment:
I get the feeling that there are too many commas in the above two sentences. Maybe split into smaller sentences? Sorry for the pedantic comment!